stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/MessageInputView.kt

```diff
@@ -438,6 +440,22 @@ public class MessageInputView : ConstraintLayout {
         binding.sendMessageButtonEnabled.isEnabled = false
     }
 
+    private fun getDisabledSendButtonIconColor(): Int {
+        return when (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK) {
+            Configuration.UI_MODE_NIGHT_NO -> R.color.stream_ui_blue
+            Configuration.UI_MODE_NIGHT_YES -> R.color.stream_ui_disabled_send_message_dark_theme
+            else -> R.color.stream_ui_blue
+        }
+    }
+
+    private fun getDisabledSendButtonIconColorStateDisabled(): Int {
+        return when (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK) {
+            Configuration.UI_MODE_NIGHT_NO -> R.color.stream_ui_grey_gainsboro
+            Configuration.UI_MODE_NIGHT_YES -> R.color.stream_ui_disabled_send_message_dark_theme
+            else -> R.color.stream_ui_grey_gainsboro
```

Comment:
Couldn't we move this to resources as well, have just one color resource that resolves to the different options based on the current configuration?